avm/res/network/firewall-policy/main.bicep

```diff
@@ -97,6 +97,12 @@ param enableTelemetry bool = true
 @description('Optional. Rule collection groups.')
 param ruleCollectionGroups array?
 
+@description('Optional. The lock settings of the service.')
```

Comment:
Thanks @krbar, great improvements.
Since new functionality is introduced, I'd suggest to update the minor version in version.json.

Reply:
@eriqua The version has been increased, thanks for pointing this out.